tests/packages/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "scripts": {
+    "test": "npm run a && npm run b && npm run c && npm run d && npm run e && npm run f && npm run g && npm run as",
+    "a": "cd ./packages/a && ../../node_modules/.bin/asc assembly/index.ts --noEmit --runtime stub --validate",
```

Comment:
Gave this another try locally and got an error when trying to directly execute the file in `.bin`. But this works:

```
  "scripts": {
    "test": "npm run a && npm run b && npm run c && npm run d && npm run e && npm run f && npm run g && npm run as",
    "a": "cd ./packages/a && node ../../../../bin/asc assembly/index.ts --noEmit --runtime stub --validate",
    "as": "cd ./packages/as && node ../../../../bin/asc as/index.ts --noEmit --runtime stub --validate",
    "b": "cd ./packages/b && node ../../../../bin/asc assembly/index.ts --noEmit --runtime stub && node ../../../../bin/asc assembly/index.ts --noEmit --runtime stub --listFiles",
    "c": "cd ./packages/c && node ../../../../bin/asc assembly/index.ts --noEmit --runtime stub --validate",
    "d": "cd ./packages/d && node ../../../../bin/asc assembly/index.ts --path packages --noEmit --runtime stub --validate --traceResolution",
    "e": "cd ./packages/d/packages/e && node ../../../../../../bin/asc assembly/index.ts --noEmit --runtime stub --validate",
    "f": "cd ./packages/d/packages/e/packages/f && node ../../../../../../../../bin/asc assembly/index.ts --noEmit --runtime stub --validate",
    "g": "cd ./packages/g && node test.js"
  }
```